src/redux/reducers/util.js

```diff
@@ -70,7 +70,10 @@ export const filterUserPayload = (user, payload) => {
   return filteredDictionaries;
 };
 
-export const filterDescriptions = (item, list) => list.filter(listItem => listItem.uuid !== item);
+export const filterItems = (item, list) => (
```

Comment:
Is this the best name for this method given what it does?
Filtering in JS is expected to return items that match something, not the other way around.

Reply:
Okay

Reply:
Thanks for that. Changed